packages/api-svc/test/unit/settlement_api_svc_route.test.ts

```diff
@@ -0,0 +1,985 @@
+    });
+
+    test("GET /matrix/:id should get SettlementMatrix by Id", async () => {
+        //Arrange
+
+        //Prepare mocked Settlement Matrix batches
+        let settlementSettlementMatrixBatches: ISettlementMatrixBatch[] = [
+            {
+                id: "DEFAULT.EUR.2023.06.21.08.30.001",
+                name: "DEFAULT.EUR.2023.06.21.08.30",
+                batchDebitBalance: "15",
+                batchCreditBalance: "15",
+                state: "OPEN",
+                batchAccounts: []
+            },
+            {
+                id: "DEFAULT.USD.2023.06.21.08.22.001",
+                name: "DEFAULT.EUR.2023.06.21.08.22",
+                batchDebitBalance: "15",
+                batchCreditBalance: "15",
+                state: "OPEN",
+                batchAccounts: []
+            },
+        ];
+
+        //Prepare mocked Matrix
+
+        let newMatrix: ISettlementMatrix = {
+            id: "TestMatrix4",
+            createdAt: new Date().getTime() / 1000,
+            updatedAt: 0,
+            dateFrom: null,
+            dateTo: null,
+            currencyCode: "EUR",
+            settlementModel: "DEFAULT",
+            batches: settlementSettlementMatrixBatches,
+            participantBalances: [],
+            participantBalancesDisputed: [],
+            state: "IDLE",
+            type: "STATIC",
+            generationDurationSecs: 2,
+            totalDebitBalance: "15",
+            totalCreditBalance: "15",
+            totalDebitBalanceDisputed: "0",
+            totalCreditBalanceDisputed: "0"
+        };
+
+        await mockMatrixRequestRepo.storeMatrix(newMatrix);
+
+        let payload: AddBatchesToMatrixCmdPayload = {
+            matrixId: "TestMatrix4",
+            batchIds: [
+                settlementSettlementMatrixBatches[0].id,
+                settlementSettlementMatrixBatches[1].id
+            ]
+        }
+
+        //Act
+        const response = await request(server)
+            .get(`/matrix/TestMatrix4`)
+            .set('authorization', AUTH_TOKEN);
+
+        //Assert
+        expect(response.status).toBe(200);
+        expect(response.body).toEqual(newMatrix);
+    });
+
+    test("GET /matrix should get list of Matrices", async () => {
+        //Arrange
+        let matrix1: ISettlementMatrix = {
+            id: "matrix1",
+            createdAt: new Date().getTime() / 1000,
+            updatedAt: 0,
+            dateFrom: null,
+            dateTo: null,
+            currencyCode: "EUR",
+            settlementModel: "DEFAULT",
+            batches: [],
+            participantBalances: [],
+            participantBalancesDisputed: [],
+            state: "IDLE",
+            type: "STATIC",
+            generationDurationSecs: 2,
+            totalDebitBalance: "15",
+            totalCreditBalance: "15",
+            totalDebitBalanceDisputed: "0",
+            totalCreditBalanceDisputed: "0"
+        };
+
+        let matrix2: ISettlementMatrix = {
```

Comment:
`const`?

Reply:
Alright @koekiebox , I will update the code. Thanks for the review.